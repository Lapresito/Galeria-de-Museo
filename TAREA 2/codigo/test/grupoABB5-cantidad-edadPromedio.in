crearGrupoABB
cantidadVisitantesGrupoABB
edadPromedioGrupoABB
crearVisitante 10 Artemio Lopez 34
agregarAGrupoABB
cantidadVisitantesGrupoABB
edadPromedioGrupoABB
crearVisitante 11 Casimira Rodriguez 22
agregarAGrupoABB
cantidadVisitantesGrupoABB
edadPromedioGrupoABB
crearVisitante 12 Zenobia Lopez 40
agregarAGrupoABB
cantidadVisitantesGrupoABB
edadPromedioGrupoABB
crearVisitante 13 Esmeralda Perez 16
agregarAGrupoABB
cantidadVisitantesGrupoABB
edadPromedioGrupoABB
liberarGrupoABB
Fin
