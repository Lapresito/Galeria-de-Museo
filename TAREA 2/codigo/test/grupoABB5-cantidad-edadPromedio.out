Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> El grupoABB ha sido creado de forma exitosa.
2> La cantidad de visitantes en el grupo es 0.
3> La edad promedio del grupo es 0.00.
4> Visitante creadox de forma exitosa.
5> Se ha agregado el visitante al grupo de forma exitosa.
6> La cantidad de visitantes en el grupo es 1.
7> La edad promedio del grupo es 34.00.
8> Visitante creadox de forma exitosa.
9> Se ha agregado el visitante al grupo de forma exitosa.
10> La cantidad de visitantes en el grupo es 2.
11> La edad promedio del grupo es 28.00.
12> Visitante creadox de forma exitosa.
13> Se ha agregado el visitante al grupo de forma exitosa.
14> La cantidad de visitantes en el grupo es 3.
15> La edad promedio del grupo es 32.00.
16> Visitante creadox de forma exitosa.
17> Se ha agregado el visitante al grupo de forma exitosa.
18> La cantidad de visitantes en el grupo es 4.
19> La edad promedio del grupo es 28.00.
20> Grupo liberado con exito.
21> Fin.
